accountClient/api.go

```diff
@@ -215,3 +217,26 @@ type InternalSigClientInfoResponse struct {
 	PublicKey  string    `json:"public_key"`
 	SigningKey string    `json:"signing_key"`
 }
+
+// InitUpdateEmailRequest wraps values needed to issue a request to start an email update
+type InitiateUpdateEmailRequest struct {
+	AccountID    string    `json:"account_id"`
```

Comment:
We should be able to determine the account id for any authenticated client server side (talk to @rosasck for some examples on that) so this value should either be optional for this request(in which case it can stay as is), or be removed from this request.

Reply:
though if you do keep it also update this type to be a uuid (@efabens I can't believe you let your mentee do this ;-) )